plugins/inputs/cloudwatch/cloudwatch.go

```diff
@@ -58,7 +68,7 @@ type (
 
 	cloudwatchClient interface {
 		ListMetrics(*cloudwatch.ListMetricsInput) (*cloudwatch.ListMetricsOutput, error)
-		GetMetricStatistics(*cloudwatch.GetMetricStatisticsInput) (*cloudwatch.GetMetricStatisticsOutput, error)
+		GetMetricData(*cloudwatch.GetMetricDataInput) (*cloudwatch.GetMetricDataOutput, error)
```

Comment:
These are wrappers to the cloudwatch client's functions. This interface is present for testing and would break if we changed that, as the cw client doesn't implement `GetMetrics`